src/components/Table/ResourceAvailabilityTable.vue

```diff
@@ -14,27 +14,7 @@
   </v-card-title>
 
   <v-data-table :headers="tableColumns" :items="resourceItem" :search="search">
-    <template v-slot:item.remove="{ item }">
-      <router-link
-        :to="{
-          name: 'Remove-Quantity',
-          params: { resourceId: item.id, userId: user.id },
-        }"
-      >
-        <v-icon color="blue">mdi-minus</v-icon>
-      </router-link>
-    </template>
 
-    <template v-slot:item.transfer="{ item }">
-      <router-link
-        :to="{
-          name: 'Transfer-Quantity',
-          params: { resourceId: item.id, userId: user.id },
-        }"
-      >
-        <v-icon color="#607D8B">mdi-swap-horizontal</v-icon>
-      </router-link>
-    </template>
```

Comment:
We are still going to be able to transfer and remove resources from an organization. Is this going to be handled some other way? Why do we remove it?

Reply:
Also this table can be reused for the resources that a user has bought I guess. In this case maybe it makes sense to remove the 2 buttons because we won't be able to remove and transfer bought resources from a user but then it shouldn't mention the orgName and should be more generic somehow. Let's discuss as I think it gets a bit complicated.